pkg/apis/networking/validation/validation.go

```diff
@@ -93,3 +98,31 @@ func ValidateNetworkPolicyUpdate(update, old *networking.NetworkPolicy) field.Er
 	allErrs = append(allErrs, ValidateNetworkPolicySpec(&update.Spec, field.NewPath("spec"))...)
 	return allErrs
 }
+
+// ValidateIpBlock validates a cidr and the except fields of an IpBlock NetworkPolicyPeer
+func ValidateIpBlock(br *networking.IpBlockRule, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	if len(br.CIDR) == 0 {
+		allErrs = append(allErrs, field.Required(fldPath.Child("cidr"), "ibBlock rules must specify a CIDR"))
```

Comment:
beyond the continued inconsistency about "ip block" vs "ip block rule", there's also a typo here ("ib")